lighthouse-core/scripts/generate-bundlephobia-database.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+/* eslint-disable no-console */
+
+// TODO(saavan) These type definitions are incorrect and need to be fixed.
+/** @typedef {import('bundle-phobia-cli').BundlePhobiaLibrary} BundlePhobiaLibrary */
+
+const fs = require('fs');
+const path = require('path');
+const getPackageVersionList = require('bundle-phobia-cli').fetchPackageStats.getPackageVersionList;
+const fetchPackageStats = require('bundle-phobia-cli').fetchPackageStats.fetchPackageStats;
+const databasePath = path.join(__dirname,
+  '../lib/large-javascript-libraries/bundlephobia-database.json');
+
+
+/** @type {Record<string, string[]>} */
+const suggestionsJSON = require('../lib/large-javascript-libraries/library-suggestions.js')
+  .suggestions;
+
+/** @type string[] */
+// @ts-ignore
+const librarySuggestions = [].concat(Object.values(suggestionsJSON), Object.keys(suggestionsJSON));
+
+
+/** @type {Record<string, any>} */
```

Comment:
we should pull out a proper typedef for this database to fix the ignores in the audit anyway.

maybe we could restructure it ever so slightly and combine with the other suggestions for something like...

```typescript
interface BundlephobiaDatabaseEntry {
  repository: string
  lastScraped: number | 'Error'
  latest: {gzip: number}
  /** A map of the version number to the size stats. Only set for the target large libraries and not the smaller suggestions. */
  versions?: {
    [version: string]: {gzip: number}
  }
}